lib/rom/sql/commands/transaction.rb

```diff
@@ -0,0 +1,17 @@
+require 'rom/commands/result'
+
+module ROM
+  module SQL
+    module Commands
+      module Transaction
+        ROM::SQL::Rollback = Class.new(Sequel::Rollback)
+
+        def transaction(options = {}, &block)
+          ::ROM::Commands::Result::Success.new(
```

Comment:
I check it and not, path is not the same because there is ROM::SQL::Commands here not ROM::Commands i can only remove those :: before ROM